app/subscriber/src/features/search-page/SearchPageContext.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { createContext, ReactNode, useContext, useState } from 'react';
+
+type SearchPageContextType = {
+  expanded: boolean;
+  setExpanded: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+const SearchPageContext = createContext<SearchPageContextType | undefined>(undefined);
+
+export function useSearchPageContext() {
```

Comment:
We need to do more of these context components to help with managing tons of child components